http.md

```diff
@@ -0,0 +1,32 @@
+# HTTP
+
+HTTP - short for Hypertext Transfer Protocol; The foundation of data communication for the World Wide Web.
+
+## Classification / acceptance questions
+
+### Apprentice aka _I know it exists_
+- I know what problem is solved by HTTP
+
+### Intermediate aka _I've been looking in the network tab_
+- I know what types of content can be transported via HTTP
+- I know the structure of a HTTP request/response
+- I can make practical use of the network tab to debug a problem
+- I know request types and response status codes
+
+### Advanced aha _?_
```

Comment:
I don't have any good suggestions but if you don't come up with any then just use:
```suggestion
### Advanced
```